README.md

```diff
@@ -121,3 +123,32 @@ addParameters({
   },
 });
 ```
+
+### react-i18next
+
+Currently, [react-i18next doesn't support other props like `locale` except `i18n`](https://react.i18next.com/latest/i18nextprovider#i-18-nextprovider-props). If you want to use i18n storybook addon, you need to wrap `I18nProvider` with `useEffect` hook.
+
+```jsx
+export function I18nProviderWrapper({ children, i18n, locale }) {
+  React.useEffect(() => {
+    i18n.changeLanguage(locale);
+  }, [i18n, locale]);
+  return <I18nProvider i18n={i18n}>{children}</I18nProvider>;
+}
+```
+
+This Provider wrapper should accept `providerLocaleKey` as props in storybook decorator parameter. If the props value corresponding to `providerLocaleKey` is changed, [we can programmatically change the language.](https://www.i18next.com/overview/api#changelanguage)
```

Comment:
Could you please also remove this line because I don't really understand what do you mean by this.

Reply:
I think there is enough said about `providerLocaleKey` in the readme so this is redundant.